span.go

```diff
@@ -47,6 +35,22 @@ type Span interface {
 	// Suitable for serializing over the wire, etc.
 	TraceContext() TraceContext
 
+	// Event() is equivalent to
+	//
+	//   Log(time.Now(), LogData{Event: event})
+	//
+	// if payload is unspecified; otherwise, only one payload argument is
+	// accepted, and Event() is equivalent to
+	//
+	//   Log(time.Now(), LogData{Event: event, Payload: payload[0]})
+	//
+	Event(event string, payload ...interface{})
```

Comment:
agreed entirely about the language aspect... annoying. your suggestion sounds fine to me, I can live with either.